h2o-algos/src/test/java/hex/tree/uplift/UpliftDRFTest.java

```diff
@@ -471,6 +473,33 @@ public void testEasyPredictMojo() throws Exception {
             Scope.exit();
         }
     }
+
+    @Test
+    public void testVariableImportance(){
```

Comment:
I saw that all the test are testing if variable importance is not null. Can we compare the values with some reference or syntetic dataset to see if the computation is correct and make sense?

Reply:
Thanks for the suggestion! You are right. I can improve the test in the next fix release. WDYT @valenad1?